src/components/ChatLog.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
+import entries from '../data/messages.json';
```

Comment:
We don't need to import the entries from the raw data anymore because it's passed in from `App` as `entries` -- the name collision also makes this add confusing!